serve.py

```diff
@@ -7,9 +7,15 @@
 import requests
 
 GIRDER_URL = os.environ.get("GIRDER_URL", "https://girder.dandiarchive.org").rstrip('/')
+
 GUI_URL = os.environ.get("GUI_URL", "https://gui.dandiarchive.org").rstrip('/')
+
 ABOUT_URL = os.environ.get("ABOUT_URL", "https://www.dandiarchive.org").rstrip('/')
 
+PUBLISH_API_URL = os.environ.get("PUBLISH_API_URL", "https://publish.dandiarchive.org/api").rstrip()
+
+JUPYTERHUB_URL = os.environ.get("JUPYTERHUB_URL", "https://hub.dandiarchive.org").rstrip()
+
```

Comment:
perhaps we just go with pre-commit (with black) across all dandi codebases. @yarikoptic @mgrauer ??

Reply:
I don't mind - better be done now while it is short code base, or not actively worked on with outstanding PRs ;-)